simple.js

```diff
@@ -1071,6 +1078,48 @@ define(function(require, exports, module) {
             });
         }
 
+
+        /**
+         * Spawns adminer and opens it on port 8082 in a new browser tab
+         */
+        function startAdminer() {
+
+            // open new browser tab
+            var tab = window.open("", "_blank");
+            if (!tab)
+                return;
+
+            tab.document.write(
+                'Starting adminer...<br>' +
+                'Please wait! This page will reload automatically.'
+            );
+
+            proc.spawn("adminer", { args: [ "--url-only" ] }, function(err, process) {
+                if (err) {
+                    console.error(err);
+                    return tab.document.write("Could not start adminer.");
+                }
+
+                // survive reloads
+                process.unref();
+
+                // read url from stdout
+                var url = "";
+                process.stdout.on("data", function handleOutput(chunk) {
```

Comment:
Is it possible a partial URL might be read via this process (and thus be broken)?

Reply:
Catching a whitespace in the end for that reason https://github.com/cs50/harvard.cs50.simple/pull/105/files/5a1f2ec4c1f1a568ca8e2ad4d972e6fda67857d4#diff-b6a68d61529250580f3e0a773a626b8eR1110.